src/gobapi/dump/to_db.py

```diff
@@ -246,6 +246,90 @@ def dump_to_db(self, full_dump=False):
         yield from self._rename_tmp_table()
         yield from self._create_indexes(model)
 
+    def create_utility_view(self):
+        """Creates view with utility columns for relating without relation table
+
+        View contains all columns from the main table, plus the RELATION_id, RELATION_VOLGNUMMER, RELATION_ref
+        and RELATION_bronwaarde columns for each RELATION (for example ligt_in_buurt_id, ligt_in_buurt_ref and
+        ligt_in_buurt_bronwaarde)
+
+        :return:
+        """
+
+        main_alias = self.model['abbreviation'].lower()
+        src_has_states = self.model.get('has_states', False)
+
+        # Collect all necessary joins and select statements
+        joins = []
+        selects = [f'{main_alias}.*']
+
+        def ref(rel_alias: str, dst_has_states: bool):
+            return f"{rel_alias}.dst_id || '_' || {rel_alias}.dst_volgnummer" \
+                if dst_has_states else f"{rel_alias}.dst_id"
+
+        for relation in [k for k in self.model['references'].keys()]:
+            # Add a join and selects for each relation
+            relation_name = get_relation_name(GOBModel(), self.catalog_name, self.collection_name, relation)
+
+            if not relation_name:
+                pass
+
+            # Determine if ManyReference and if destination has states
+            src_field = self.model['all_fields'].get(relation)
+            dst_catalog_name, dst_collection_name = src_field['ref'].split(':')
+            dst_has_states = GOBModel().has_states(dst_catalog_name, dst_collection_name)
+            is_many = src_field['type'] == 'GOB.ManyReference'
+
```

Comment:
== fully_qualified_type_name(GOB.ManyReference)

Reply:
Thanks!


.. It thought, there should be a function for this repeated pattern.. and then I forgot.. it appears I've thought that before